### ISSUE TITLE
Unexpected "exclude: /node_modules/" behaviour

### ISSUE THREAD

**Issue by TheImpressionist** [webpack-contrib/css-loader#981]
- Operating System: macOS Catalina 10.15
- Node Version: 12.7.0
- NPM Version: 6.10.0
- webpack Version: 4.39.1
- css-loader Version: 3.1.0

### Expected Behavior

Using `exclude: /node_modules/` would exclude unneeded CSS files found under the `node_modules` folder.

### Actual Behavior

Throws the following error, saying I need an appropriate loader for CSS.

```
ERROR in ./node_modules/antd/lib/tooltip/style/index.css 4:0
Module parse failed: Unexpected token (4:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|/* stylelint-disable declaration-bang-space-before */
| /* stylelint-disable declaration-bang-space-before */
> .ant-tooltip {
|   position: absolute;
|   z-index: 1060;
ℹ ｢wdm｣: Failed to compile
```

### Code

The issue is caused here. I tried removing any other loaders. You can reproduce this with just a minimal setup, which is listed after my CSS configuration.

```js
{
    test: /\.(s?)css$/i,
    exclude: /node_modules/,
    use: [
      !shouldExtractStyles() ? 'style-loader' : MCEP.loader,
      'css-loader',
      {
        loader: 'postcss-loader',
        options: {
          plugins: [
            require('autoprefixer'),
            require('cssnano')({
              zindex: false,
              reduceIdents: false,
            })],
        },
      },
      'sass-loader',
    ],
  }
```

Minimal reproducible:

```js
  {
    test: /\.css$/i,
    exclude: /node_modules/,
    use: ['style-loader', 'css-loader']
  }
```

### How Do We Reproduce?

Just try and exclude the `/node_modules/` folder.

Unless I'm completely failing to grasp the purpose of `exclude` the code should just exclude the files and not try to bundle them into my CSS. However, I get an error saying that it cannot understand what the file is and may need an appropriate loader. However, if I leave out `exclude` all together it works just fine, though it includes a lot of useless CSS which is ruining a lot of things for us.

**Comment by evilebottnawi** [comment]
Not related to css loader, it is part of webpack api

**Comment by yveshema** [comment]
I understand this has been closed for a while but I'm not sure an answer was provided.

I encountered a similar issue, except that in my case instead of an `exclude` clause, I had an `include` clause specifying the `src` folder where all my css files were located. 

What I could not understand is that these clauses appear to work fine with other loaders such as the `babel-loader`, so I expected similar behavior with the `css-loader`. 

My setup was the following:

```
{
   test: /\.css$/i,
   use: ['style-loader', 'css-loader'],
   include: path.resolve(__dirname, 'src'),
}
```
